.changesets/remove-support-for-versions-in-eol.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
```

Comment:
This is a major version bump as we remove compatibility with older Ruby versions.

Reply:
I can see it as a minor bump instead. We're just being explicit about the removal of support for versions that went into EoL a long time ago.

Reply:
Minor version bump is fine.